parser/parser.go

```diff
@@ -27,6 +27,7 @@ const (
 type Rule struct {
 	RuleFor      string
 	CannotImport []string
+	CanImport    []string
```

Comment:
Could you update the `logging` package to also print the allowed imports when the `-verbose` env var is set?